source/js/components/ExpandedRecipeCard.js

```diff
@@ -1,238 +1,751 @@
+    ratingLiArr[1].innerHTML = getNumLikes(data) + " likes.";
+
+    // Set image:
+    const recImg = this.shadowRoot.querySelector("section#header > img");
+    const recImgSrc = getImgSrc(data);
+    recImg.setAttribute("src", recImgSrc);
+    recImg.setAttribute("alt", title); // set alt text for rec image to rec's title
+
+    // Set description:
+    const descrEl = this.shadowRoot.querySelector("div > p.description");
+    const descr = getSummary(data);
+    descrEl.innerHTML = descr;
+
+    // Set/create ingredients list
+    // (create list items, <li>'s, for individual ingredients
+    // and their amounts):
+    const ingUlEl = this.shadowRoot.querySelector("div > ul.ingredients");
+
+    const ingredArr = getIngredArray(data); // array of ingredients 
+
+    for(let i = 0; i < ingredArr.length; i++) {
+      let ingredListItem = document.createElement("li");
+      ingredListItem.setAttribute("class", "item");
+      ingredListItem.innerHTML = ingredArr[i];
+
+      ingUlEl.appendChild(ingredListItem);
     }
 
-    div.rating--wrapper {
-      align-items: center;
-      column-gap: 5px;
-      display: flex;
-      justify-items: center;
-      margin-top: 10px;
+    // Set/create quick facts list
+    // (create list items, <li>'s, for individual quick facts
+    // and their amounts):
+    const qfUlEl = this.shadowRoot.querySelector("ul.facts"); // qf list <ul>
+    // console.log("qfUlEl: " + qfUlEl);
+    const qfArr = getQuickFactsArr(data); // get quick facts array
+    // console.log("qfArr: " + qfArr);
+    for(let i = 0; i < qfArr.length; i++) {
+      let qfListItem = document.createElement("li");
+      qfListItem.setAttribute("class", "item");
+      qfListItem.innerHTML = qfArr[i];
+      qfUlEl.appendChild(qfListItem);
     }
     
-    div.rating--wrapper > img {
-      height: auto;
-      display: inline-block;
-      object-fit: scale-down;
-      width: 78px;
+    // Set/create steps list
+    // (create list items, <li>'s, for individual steps):
+    const stepOlEl = this.shadowRoot.querySelector('div.sub-body > ol.steps'); // step list <ol>
+    // console.log("stepOlEl: " + stepOlEl);
+    const stepsArr = getRecInstructionsArr(data);
+    // console.log("stepsArr: " + stepsArr);
+    for(let i = 0; i < stepsArr.length; i++) {
+      let stepListItem = document.createElement("li");
+      stepListItem.setAttribute("class", "steps");
+      stepListItem.innerHTML = stepsArr[i];
+      stepOlEl.appendChild(stepListItem);
     }
+    
+  } 
 
-    header {
-      align-items: flex-start;
-      column-gap: 10px;
-      display: grid;
-      grid-template-areas:
-        'title title img'
-        'meta meta img'
-        'desc desc img';
+    // return data of current expanded recipe:
+    get data() {
+      return this.json;
     }
 
-    header p {
-      margin: 0;
-    }
+  } // end of class
+  customElements.define('recipe-expand', RecipeExpand);
```

Comment:
I don't think defining a custom html element is necessary.  @AntoniaY31 and @fjontran had already completed a recipe-info.html (expanded recipe page), which we can incorporate directly into index.html, and populate directly.

Reply:
this is already in the main branch

Reply:
Ah I see. Unfortunately, it looks like the html was already created. I'll get back to you on what we'll do with this after today's SCRUM meeting.